Libraries/Components/Button.js

```diff
@@ -25,107 +25,210 @@ import type {ColorValue} from '../StyleSheet/StyleSheet';
 
 type ButtonProps = $ReadOnly<{|
   /**
-   * Text to display inside the button
+    Text to display inside the button. On Android the given title will be
+    converted to the uppercased form.
    */
   title: string,
 
   /**
-   * Handler to be called when the user taps the button
+    Handler to be called when the user taps the button. The first function
+    argument is an event in form of [PressEvent](pressevent).
    */
   onPress: (event?: PressEvent) => mixed,
 
   /**
-   * If true, doesn't play system sound on touch (Android Only)
-   **/
+    If `true`, doesn't play system sound on touch.
+
+    @platform android
+
+    @default false
+   */
   touchSoundDisabled?: ?boolean,
 
   /**
-   * Color of the text (iOS), or background color of the button (Android)
+    Color of the text (iOS), or background color of the button (Android).
+
+    @default {@platform android} '#2196F3'
+    @default {@platform ios} '#007AFF'
    */
   color?: ?ColorValue,
 
   /**
-   * TV preferred focus (see documentation for the View component).
+    TV preferred focus.
+
+    @platform tv
+
+    @default false
    */
   hasTVPreferredFocus?: ?boolean,
 
   /**
-   * TV next focus down (see documentation for the View component).
-   *
-   * @platform android
+    Designates the next view to receive focus when the user navigates down. See
+    the [Android
+    documentation](https://developer.android.com/reference/android/view/View.html#attr_android:nextFocusDown).
```

Comment:
@motiz88 I have used a `vscode` extension called `rewrap` to wrap the comments to markdown syntax. Some lines won't wrap to a maximum width(because they do not have a white space). The markdown doesn't wrap these lines either so in my opinion we should keep them as it is. What's your opinion?